orcoursetrion/config.py

```diff
@@ -24,6 +24,12 @@
     # GitHub API URL
     ('ORC_GH_API_URL', 'https://github.mit.edu/api/v3/'),
 
+    # GitHub Committer Name
+    ('ORC_GH_NAME', 'Orcoursetrion ODL'),
+
+    # GitHub Committer Email
+    ('ORC_GH_EMAIL', 'orcoursetrion@example.com'),
```

Comment:
This is really out of the scope of this review, but was there a reason that these config defaults are being stored as a list of tuples rather than a dict?